blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+
+<div style="padding-left: 2rem">
+
+[**🤖 Introducing Automation: first stages of campaign management**](#introducing-automation-first-stages-of-campaign-management)<br />
+TO DO???
+
+[**🧠 Beta release of fast and precise LSIF-based code intelligence**](#beta-release-of-fast-and-precise-lsif-based-code-intelligence)<br />
+Faster, more precise code intelligence for TypeScript, Go, Python, and C/C++
+
+[**🔎 Search result tabs for improved resuslt type discoverability**](#search-result-tabs-for-improved-resuslt-type-discoverability)<br />
+
+[**📊 Out of the box monitoring of your Sourcegraph instance**](#out-of-the-box-monitoring-of-your-sourcegraph-instance)<br />
+
+[**📉 Behind the scenes: Even more benchmarking, automated testing, and QA**](#behind-the-scenes-even-more-benchmarking-automated-testing-and-qa)<br />
+
+[**🔦 Quickly toggle the browser extension on/off**](#quickly-toggle-the-browser-extension-onoff)<br />
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community.
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Introducing Automation: first stages of campaign management
+
+TO DO: Include one of the demo videos here?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+[Large scale code change Automation](https://about.sourcegraph.com/product/automation) is coming soon to Sourcegraph. With this functionality, you will be able to automate large-scale code changes to remove legacy code, fix critical security issues, and pay down tech debt. We have started our implementation with campaign management - the ability to track sets of changes (e.g. pull requests) to ensure they are rolled out throughout your organization.
```

Comment:
Since we aren't announcing anything specifically in this release, I think we should just focus on the vision (not the intermediate milestones).

```suggestion
We have started working to add new [Automation](https://about.sourcegraph.com/product/automation) workflows to Sourcegraph. Soon, you will be able to automate large-scale code changes to remove legacy code, fix critical security issues, and pay down tech debt across all of your repositories.
```